# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1244827765 14400
#      Fri Jun 12 13:29:25 2009 -0400
# Node ID 70e0d3b1f26f85bce277a3fb2d4e5b9047731fab
# Parent  2ca8879a140c90cf489871788451222338914f84
fix typos in previous change

diff --git a/src/ov-fcn-handle.cc b/src/ov-fcn-handle.cc
--- a/src/ov-fcn-handle.cc
+++ b/src/ov-fcn-handle.cc
@@ -1317,29 +1317,29 @@ make_fcn_handle (const std::string& nm)
 	  tnm = "mtimes";
 	  break;
 
 	case '/':
 	  tnm = "mrdivide";
 	  break;
 
 	case '^':
-	  tnm = 'mpower';
+	  tnm = "mpower";
 	  break;
 
 	case '\\':
 	  tnm = "mldivide";
 	  break;
 
 	case '<':
 	  tnm = "lt";
 	  break;
 
 	case '>':
-	  tnm = "glt";
+	  tnm = "gt";
 	  break;
 
 	case '&':
 	  tnm = "and";
 	  break;
 
 	case '|':
 	  tnm = "or";
@@ -1378,17 +1378,17 @@ make_fcn_handle (const std::string& nm)
 %!      "\'", "ctranspose";
 %!      "+", "plus";
 %!      "-", "minus";
 %!      "*", "mtimes";
 %!      "/", "mrdivide";
 %!      "^", "mpower";
 %!      "\\", "mldivide";
 %!      "<", "lt";
-%!      ">", "glt";
+%!      ">", "gt";
 %!      "&", "and";
 %!      "|", "or"};
 %! for i = 1:rows (x)
 %!   assert (functions (str2func (x{i,1})).function, x{i,2})
 %! endfor
 */
 
 DEFUN (functions, args, ,
